python/setup.cfg

```diff
@@ -1,11 +1,35 @@
 # Copyright (c) 2018, NVIDIA CORPORATION.
 
 [flake8]
-exclude = cuML,ml-prims,__init__.py,versioneer.py
+filename = *.py, *.pyx, *.pxd
+exclude = 
+   cpp,
+   thirdparty,
+   versioneer.py,
+   *.egg,
+   build/,
+   # docs,
```

Comment:
should probably add this back to exclude list?

Reply:
Good catch. Done